frontend/components/Media.vue

```diff
@@ -193,21 +230,114 @@ export default {
         })
         .show()
     },
+    handleYoutube(e, media) {
+      require('basiclightbox/dist/basicLightbox.min.css')
+      const basicLightbox = require('basiclightbox')
+      basicLightbox
+        .create(
+          `<iframe
+        width="100%"
+        src="https://www.youtube.com/embed/${this.getYoutubeId(media.url)[2]}"
+        frameborder="0"
+        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        class="frame-video"
+      ></iframe>`,
+          {
+            closable: true
+          }
+        )
+        .show()
+    },
+    handleVimeo(e, media) {
+      require('basiclightbox/dist/basicLightbox.min.css')
```

Comment:
Any reason to require here, not at the top of file?

Reply:
I actually didnt u can import it on the top of the file, thanks for this!